package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "frosting",
+  "version": "1.0.0",
+  "description": "Every cake needs a little frosting! Add some flair to your project\r by adding some Frosting style.",
+  "main": "dist/index.css",
+  "scripts": {
+    "build": "parcel build index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kbaumzie/frosting.git"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
I would recommend updating some of these fields, you can look at other `package.json` files in other projects on github to see some common configurations, or look at the npm package.json docs